src/components/graph/menus/modification-list-item.js

```diff
@@ -73,6 +73,8 @@ export const ModificationListItem = ({
             return modif.lineToAttachToId;
         } else if (modif.type === 'LINES_ATTACH_TO_SPLIT_LINES') {
             return modif.attachedLineId;
+        } else if (modif.type === 'GENERATOR_SCALING') {
+            return modif.uuid;
```

Comment:
displaying the modification uuid is useless for the user ... 
Maybe we should set a name in the modification creation dialog ?